query/command/command.go

```diff
@@ -273,9 +273,10 @@ func (cmd *SelectCommand) Execute(context ExecutionContext) (CommandResult, erro
 	case result := <-results:
 		lists := make([]api.SeriesList, len(result))
 		for i := range result {
-			lists[i], err = result[i].ToSeriesList(evaluationContext.Timerange, cmd.Expressions[i].QueryString())
-			if err != nil {
-				return CommandResult{}, err
+			var convErr *function.ConversionFailure
```

Comment:
You can't use `:=` when one of the things at the left is an index (like `lists[i]`).